src/crl.c

```diff
@@ -364,6 +364,20 @@ int CheckCertCRL(WOLFSSL_CRL* crl, DecodedCert* cert)
     }
 #endif
 
+#if defined(OPENSSL_ALL) && defined(WOLFSSL_CERT_GEN) && \
```

Comment:
LoadCertByIssuer() depends on methods, such as wc_EncodeName() and ConvertNIDToWolfSSL, that have been gated by CERT_GEN, CERT_REQ and/or CERT_EXT. Therefore, I have gated LoadCertByIssuer() and calling point by those macro rather than I remove those guard.